tests/test_jobs.py

```diff
@@ -2428,6 +2428,24 @@ def test_dagrun_root_fail_unfinished(self):
         dr_state = dr.update_state()
         self.assertEqual(dr_state, State.RUNNING)
 
+    def test_dagrun_root_after_dagrun_unfinished(self):
+        """
+        DagRuns with one successful and one future root task -> SUCCESS
+        """
+        dag_id = 'test_dagrun_states_root_future'
+        dag = self.dagbag.get_dag(dag_id)
+        dag.clear()
+        scheduler = SchedulerJob(dag_id, num_runs=2)
+        # we can't use dag.run or evaluate_dagrun because it uses BackfillJob
+        # instead of SchedulerJob and BackfillJobs are allowed to not respect start dates
```

Comment:
nit: you mean `unlike SchedulerJob and BackfillJobs are allowed to not respect start dates` ?

Reply:
@dima-asana , WDYT?